kafka/consumer/Dockerfile

```diff
@@ -4,13 +4,14 @@
 # --------------------------------------------------------------------------------------------
 
 FROM golang:1.21 as builder
```

Comment:
If we're already modifying this file, we should consider switching this to golang:1.22.  Also, we need to use an mcr mirror for this.

Reply:
I actually want to change the Dockerfile back to what it was before once this PR is merged so that it does not rely on the local util. It will be able to pull from remote github module.